### ISSUE TITLE
(Enhancement) More settings

### ISSUE THREAD

**Issue by 75h7** [GoofGarage/Win11Clean#3]
Some more settings that could be included:

1. Settings > Time and Language > Typing > Typing Insights off
2. Settings > Accessibility > Keyboard > Sticky Keys/Filter Keys off
3. Windows Security > App & browser control > [Exploit protection](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/enable-exploit-protection?view=o365-worldwide) settings > change from "use default" to "on by default" for the ones that are "use default (on)"
4. Windows Security > Device security > Core isolation > Memory integrity on
5. On the far right of the taskbar by the time, notifications will show up, turn off notifications for "suggested"
6. File explorer > Home (or any folder) > Folder options > General > Privacy > uncheck "show frequently used folders" and "show files from office.com" and clear history

recommend [privacy.sexy](https://privacy.sexy/)

**Comment by GoofGarage** [comment]
Earmarking these suggestions for the October release.

**Comment by GoofGarage** [comment]
Item 1 was addressed as a potential attack vector and Section 3.4.2 was created to address it.

Item 2 does not have privacy or security implications.  Sticky Keys/Filter Keys is an accessibility function that's been around since at least Windows XP (if not Windows 2000).  Users are always prompted with how to disable it whenever it is first triggered.

Item 3 was added in Section 5.1.1.

Item 4 is useful, but has very serious implications for developers who use virtual machines.  Personally I cannot enable it or it actively prevents me from doing client development work, and as a result I won't blanket recommend it by default to others.

Item 5 was already covered by Section 3.2.3, "Post-installation: Disable ads shown as notifications" in previous releases of the guide.

Item 6 was added as additional suggestions for power users in Section 3.2.2, "Post-installation: Disable ads in the File Explorer"
